api/src/main/java/com/lightbend/lagom/javadsl/api/broker/Subscriber.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.api.broker;
+
+import java.util.concurrent.CompletionStage;
+
+import akka.Done;
+import akka.stream.javadsl.Flow;
+import akka.stream.javadsl.Source;
+
+/**
+ * A Subscriber for consuming messages from a message broker.
+ *
+ * @param <Message> The message type.
+ *
+ * @note This class is not meant to be extended by client code.
+ */
+public interface Subscriber<Message> {
+
+  /**
+   * Subscribers with the same group id belong to the same group.
+   * It is guaranteed that a set of subscribers with the same group id 
+   * acts as if it was a single logical subscriber, i.e., each message 
+   * is processed only by a single subscriber in the group.   
```

Comment:
I think the group concept can be confusing to new users, that are not familar with Kafka. Perhaps steal some of the explanation from the Kafka docs http://kafka.apache.org/0100/javadoc/index.html?org/apache/kafka/clients/consumer/KafkaConsumer.html

> To get semantics similar to a queue in a traditional messaging system all processes would be part of a single consumer group and hence record delivery would be balanced over the group like with a queue. Unlike a traditional messaging system, though, you can have multiple such groups. To get semantics similar to pub-sub in a traditional messaging system each process would have its own consumer group, so each process would subscribe to all the records published to the topic.